site/docs/theming/index.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Theming
+layout: DetailTechnical
+sidebar:
+  label: Overview
+  priority: 1
+---
+
+The Salt design system enables teams to modify design variables and theme projects to their requirements. Understanding how Salt structures and manages these design variables will help you create consistent themes for your project.
+
+## What a theme is
+
+A theme is a controlled change (or series of changes) to the design system's default design language. The goal is to change the system's look and feel to better fit with alternative established guidelines.
+
+## When to theme
+
+Salt offers a consistent design language ready for teams to use in their projects. Theming is not mandatory, but it can help a team meet their project's requirements.
+
+Before theming, consider these three questions:
+
+1. Is the default Salt design language unsuitable for my project?
+2. Do I have a set of alternative design guidelines to follow?
+3. Does my project need to follow those guidelines?
+
+If the answer to all three is "yes," theming may be suitable for your project.
+
+<ExampleContainer type="positive" customPillText="Best practice">
+  Use theming when your project needs to follow alternative design guidelines.
+</ExampleContainer>
+
+## How to theme
+
+Theming should aim to maintain a consistent look and feel. Salt is designed to ensure visual consistency, and understanding how Salt achieves this is key to producing consistent themes.
+
+While users can modify individual components to meet their needs, this can create inconsistencies as users have to remember to apply changes where relevant to other components. Salt's tokens make it easier to apply changes while maintaining consistency.
+
+### What a token is
+
+A token stores design decisions. Multiple components can use the same token to ensure a given design decision applies consistently across the design system.
+
+An example of a token is `--salt-status-error-foreground`. This token is used to set the foreground color when a component communicates an error message. That means both `Dialog` and `Toast` can use this token to set the foreground color when necessary.
```

Comment:
I think it would be worth explaining what foreground is (ie., it's predominately the text and icon elements sitting against a component background).